aws-datastore/src/test/java/com/amplifyframework/datastore/AWSDataStorePluginConfigurationTest.java

```diff
@@ -37,68 +33,32 @@
 @RunWith(RobolectricTestRunner.class)
 public final class AWSDataStorePluginConfigurationTest {
     /**
-     * If no configuration is specified, it's okay, sync mode will default to
+     * If configuration is null, it's okay, sync mode will default to
      * {@link AWSDataStorePluginConfiguration.SyncMode#LOCAL_ONLY}.
```

Comment:
As a matter for another PR (its unrelated to this PR) - this behavior is backwards :-(

Reply:
This PR also contains the logic to make AWS DataStore plugin's configuration parameter to be nullable, so I felt like including this test was necessary.